### PR TITLE
fix(#140): general technical debt

### LINKED ISSUE #146: Facet Filter
We now currently return data for searches, develop a facet around return data samples.

### PR DESCRIPTION
**Pull request by tholulomo**
These updates are temporary updates that require additional TD tickets for test fixes and code refactoring.

The sample page needs a redo including its unit testing. Additionally, I am writing out a few TD tickets for some of the new issues I noticed while reviewing and working on this TD ticket. Also, new testing is required for components that originally do not have unit testing attached at dev time.

closes #146

### CODE REVIEW

File: app/src/components/explorer/SearchResultsTable.vue [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+              </div>
+            </div>
+          </md-card-header>
+        </div>
+      </div> -->
+      <!-- <pagination
+        :cpage="page"
+        :tpages="totalPages"
+        @go-to-page="loadItems($event)"
+      /> -->
```
Comment (stouffers) [inline app/src/components/explorer/SearchResultsTable.vue:231]: we should delete this instead of commenting out

**Review by stouffers — APPROVED** [review]

File: app/Dockerfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ RUN npm install
 COPY . .
```
[the remark this answers is not in the document]
Comment (tholulomo) [reply app/Dockerfile:5]: I'll create a TD ticket and add the dockerignore.

File: app/src/components/explorer/SearchResultsTable.vue (hunk 1 above)
Comment (tholulomo) [reply app/src/components/explorer/SearchResultsTable.vue:231]: That content is still needed but we don't have a backend logic for it yet, hence the commenting out. The getCharts there is just a place-holder.

File: app/Dockerfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (tholulomo) [reply app/Dockerfile:5]: See here https://github.com/Duke-MatSci/materialsmine/issues/155

**Review by aswallace — APPROVED** [review]
Was not able to run the changes locally, but seems to be an issue with my computer locally

File: app/src/pages/explorer/FacetFilter.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                    <div class="md-layout-item md-size-50">
+                        <div>
+                            <h2 class="sample-page_header">Definition (MaterialsMine):</h2>
+                            <p>{{ selectedFacetFilterMaterials.parsedResponseDefinition[0] ?
+                                selectedFacetFilterMaterials.parsedResponseDefinition[0]['definition'] : 'N/A' }}</p>
+                        </div>
+                        <div v-if="selectedFacetFilterMaterials" data-test="materialComponents">
+                            <h2 class="sample-page_header">Curated Research Articles:</h2>
+                            <p>There are currently <strong>{{ selectedFacetFilterMaterials.parsedResponseCount[0] ?
+                                selectedFacetFilterMaterials.parsedResponseCount[0]['SampleCount'] : 0 }}</strong> nanocomposite samples in the MaterialsMine knowledge graph that contain Strain Rate as a reported attribute or property.</p>
```
[the remark this answers is not in the document]
Comment (tholulomo) [reply app/src/pages/explorer/FacetFilter.vue:20]: No, typo

**Pull request merged**